docs/REQUIREMENTS.md

```diff
@@ -10,38 +10,38 @@ Table of Contents
     - [VMware vSphere](#vmware-vsphere)
       - [Resources](#resources)
       - [Machine Template Requirements](#machine-template-requirements)
-    - [Bare-Metal](#bare-metal)
+    - [Physical Machines](#physical-machines)
   - [Network](#network)
     - [CIDR Block](#cidr-block)
     - [Static IP Addresses](#static-ip-addresses)
     - [Floating IP Addresses](#floating-ip-addresses)
   - [Examples](#examples)
-    - [vCenter/vSphere Sample `tfvars` File](#vcentervsphere-sample-tfvars-file)
-    - [Bare Metal Sample `inventory` File](#bare-metal-sample-inventory-file)
+    - [vCenter/vSphere Sample tfvars File](#vcentervsphere-sample-tfvars-file)
+    - [Physical Machine Sample Inventory File](#physical-machine-sample-inventory-file)
   - [Deployment](#deployment)
   - [Third-Party Tools](#third-party-tools)
 
 ## Operating System
 
-An Ubuntu Linux operating system is required for the tasks associated with standing up infrastructure using the tools in this repository.
+An Ubuntu Linux operating system is required for the machine that uses the tools in this repository to perform the tasks associated with standing up infrastructure for a SAS Viya deployment. 
 
-| OS | Description |
+| Operating System | Description |
 | --- | --- |
 | [Ubuntu 20.04 LTS](https://releases.ubuntu.com/20.04/) | You must have a user account and password with privileges that enable unprompted `sudo`. You must also have a shared "private/public" SSH key pair to use with each system. These are required for the Ansible tools that are described below. |
```

Comment:
The base version of Ubuntu for this release is 22.04 Need to alter any references to 20.04 -> 22.04